.github/workflows/tests.yml

```diff
@@ -151,27 +151,30 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v2
       with:
-        python-version: 3.7
+        python-version: 3.9
     - uses: actions/cache@v1
       with:
         path: ~/.cache/pip
         key: ${{ runner.os }}-pip-${{ hashFiles('**/setup.py') }}-${{hashFiles('**/requirements.txt')}}
         restore-keys: |
           ${{ runner.os }}-pip-
+
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install jupyterlab==3.0.3
+        python -m pip install -U jupyterlab~=3.2 jupyter-packaging~=0.10
 
-    - name: Use Node.js 12.x
-      uses: actions/setup-node@v1
+    - name: Set up Node
+      uses: actions/setup-node@v2
       with:
-        node-version: 12.x
+        node-version: 14.x
```

Comment:
Is Node 12 no longer supported?

Reply:
It could be that node 12 was causing issues for this workflow. Although other workflows still use that version.

We could try to switch back to 12 and see if the check fails.

Reply:
The CI check failed after switching to node 12 in this commit: https://github.com/jupyter-widgets/ipywidgets/runs/4313698147?check_suite_focus=true

With:

![image](https://user-images.githubusercontent.com/591645/143266094-1bade993-3a04-4c95-9854-138e684f622b.png)

And passes after switching back to node 14: https://github.com/jupyter-widgets/ipywidgets/runs/4313796358?check_suite_focus=true